src/components/select-field/BaseSelectField.js

```diff
@@ -66,14 +70,22 @@ type State = {
     shouldScrollIntoView: boolean,
 };
 
-export const OVERLAY_SCROLLABLE_CLASS = 'bdl-SelectField-overlay--scrollable';
+function defaultOptionRenderer({ displayText }: SelectOptionProp) {
```

Comment:
Maybe move this out to its own file with its own mini test

Reply:
It would make sense to move it out, but not sure if it is going to be used anywhere else.